pallets/duster/src/lib.rs

```diff
@@ -91,7 +91,8 @@ pub mod pallet {
 		type Erc20Support: hydradx_traits::evm::Erc20Inspect<Self::AssetId>
 			+ hydradx_traits::evm::Erc20OnDust<Self::AccountId, Self::AssetId>;
 
-		/// Duster for accounts with AToken dusts
+		/// Whitelist for dust removal, used to check if an account should be excluded from dusting
+		type DustRemovalWhitelist: Contains<Self::AccountId>;
```

Comment:
the naming and description is bit confusing and feels like doing same thing somewhere else. 

i would suggest to call it differently, eg  ExtendedWhitelist